dashboard/assets/packages/ui/src/typings/xk6-dashboard-model.d.ts

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2023 Raintank, Inc. dba Grafana Labs
```

Comment:
I would be interested to know if the class would allow generics so that we don't have to do this.

https://www.tutorialsteacher.com/typescript/typescript-generic-class